### ISSUE TITLE
Define hyperparameters for the training of the models

### ISSUE THREAD

**Issue by vollOlga** [GeroVanMi/algorithmic-quartet-mlops#21]
- [ ] Define the Hyperparameters that influence the model performance. (i.e. Learning Rate, Batch Size, etc.) 
- [ ] Track the Hyperparameters per Run in W&B.

**Comment by GeroVanMi** [comment]
@vollOlga Maybe I misunderstand this issue, but isn't that part of the training pipeline? This is not related to metrics or deployment, right? If so, I would close this again.

**Comment by vollOlga** [comment]
yes, you're right, it's more for training, but we can also set it up as a wall for experimental settings

**Comment by GeroVanMi** [comment]
@vollOlga 
What do you mean by "a wall for experimental settings"?

**Comment by vollOlga** [comment]
Diese Parameter in den Experimenten auf dem Wandb auflisten

**Comment by GeroVanMi** [comment]
Ah! I see, you mean that we could track these Hyperparameters on W&B! Yes, that's a very good idea! 

@Patrickliuu I am assigning this to you, this is an issue where we can track all of the parameters that are relevant for the model training (i.e. Learning Rate, Batch Size, + all the specific params for the GAN) 

Once we start building the training pipeline, we should make sure that these parameters are linked to the runs, so that we can correlate the model performance with the hyperparameters. Hence, I set this to the MVP milestone, since we don't need to talk about this for the pitch next week.

**Comment by GeroVanMi** [comment]
I believe we just stick with the original hyperparameters from the HuggingFace author.
